README.md

```diff
@@ -14,7 +14,7 @@ Add the gem to your Gemfile's development section.
 
 ```ruby
 group :development, :test do
-  gem 'devtools', '~> 0.1.1'
+  gem 'devtools', '~> 0.1.x'
 end
```

Comment:
Lets simply kill the whole Gemfile example instead and say: Install the gem `devtools` in development environment via your preferred method.

I think this is more exact / universal than giving a copy & pastable code example we have to maintain. And the target audience of devtools will likely not get confused.